overlord/state/task.go

```diff
@@ -51,6 +51,8 @@ type Task struct {
 
 	spawnTime time.Time
 	readyTime time.Time
+
+	scheduledTime time.Time
```

Comment:
Can we name this after idea of _at_ time, borrowing from the old school `at` tool?

Suggested API:
```
task.At(when)          // previously st.ScheduleTask(task, when)
task.AtTime() => when  // we have ReadyTime(), SpawnTime() already 

... `json:"at-time"`

etc
```